picard/coverart/image.py

```diff
@@ -117,7 +117,14 @@ class CoverArtImage:
     is_front = None
     sourceprefix = "URL"
 
-    def __init__(self, url=None, types=[], comment='', data=None):
+    def __init__(self, url=None, types=[], comment='', data=None,
+                 from_file=None):
```

Comment:
It took me a while to figure out that `from_file` is the filename (so a string) and not a simple boolean. Maybe rename it?

I also wonder why `CoverArtImage` needs to know about the possibility of images from files instead of a new LocalCoverArtImage (or something like that) class since we already have {Caa,CaaThumbnail,Tag}CoverArtImage subclasses?